### ISSUE TITLE
The  Volatile GPU-Util 0%

### ISSUE THREAD

**Issue by zjuzxm** [MhLiao/DB#50]
Dear author:
       When I trained the model, I found the Volatile GPU-Util equal to 0%. Is anything wrong with my method?
+-------------------------------+----------------------+----------------------+
|   8  TITAN Xp            On   | 00000000:8E:00.0 Off |                  N/A |
| 23%   32C    P2    65W / 250W |   1683MiB / 12196MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|   9  TITAN Xp            On   | 00000000:8F:00.0 Off |                  N/A |
| 23%   34C    P2    60W / 250W |   1685MiB / 12196MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|  10  TITAN Xp            On   | 00000000:90:00.0 Off |                  N/A |
| 23%   31C    P2    61W / 250W |   1685MiB / 12196MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+
|  11  TITAN Xp            On   | 00000000:91:00.0 Off |                  N/A |
| 23%   31C    P2    60W / 250W |   1693MiB / 12196MiB |      0%      Default |
+-------------------------------+----------------------+----------------------+

**Comment by MhLiao** [comment]
@zjuzxm Is the GPU-Util always 0%? How many images do your training data contain? You can check whether it is blocked in the [dataloader](https://github.com/MhLiao/DB/blob/fc84af89f5c9dc8e92cdc3a8b0b6a8af9bd8fce1/data/image_dataset.py#L35).

**Comment by zjuzxm** [comment]
I checked the log, it seemed like the there is something wrong with the training. I used 4 Titian GPUs to train the total_text with command
`python train.py experiments/seg_detector/totaltext_resnet18_deform_thre.yaml --num_gpus 4`
But the GPU-Util is always 0%, and the log are as follows.
`[INFO] [2019-12-10 07:59:42,832] Training epoch 0
[INFO] [2019-12-10 07:59:47,038] Training epoch 0
[INFO] [2019-12-10 07:59:59,624] Training epoch 0
[INFO] [2019-12-10 07:59:59,747] Training epoch 0
[INFO] [2019-12-10 07:59:59,867] Training epoch 0
[INFO] [2019-12-10 08:00:11,334] Training epoch 0
[INFO] [2019-12-10 08:00:16,321] Training epoch 0
[INFO] [2019-12-10 08:02:45,156] Training epoch 0
[INFO] [2019-12-10 08:18:24,331] step:      0, epoch:   0, loss: 40.107483, lr: 0.007000
[INFO] [2019-12-10 08:18:24,347] bce_loss: 7.067471
[INFO] [2019-12-10 08:18:24,348] thresh_loss: 0.953832
[INFO] [2019-12-10 08:18:24,349] l1_loss: 0.381630
[INFO] [2019-12-10 08:37:19,139] Training epoch 1
[INFO] [2019-12-10 09:20:08,563] Training epoch 2
[INFO] [2019-12-10 10:05:06,779] Training epoch 3
[INFO] [2019-12-10 10:54:39,408] Training epoch 4
[INFO] [2019-12-10 11:32:45,776] Training epoch 5
[INFO] [2019-12-10 11:49:39,380] step:    450, epoch:   5, loss: 3.489134, lr: 0.006974
[INFO] [2019-12-10 11:49:39,422] bce_loss: 0.450540
[INFO] [2019-12-10 11:49:39,426] thresh_loss: 0.402760
[INFO] [2019-12-10 11:49:39,439] l1_loss: 0.083367
[INFO] [2019-12-10 11:50:02,397] Training epoch 6`

**Comment by MhLiao** [comment]
@zjuzxm Your training speed is too slow. It only takes about 1\~2 minutes/epoch for me to train the total-text dataset.

**Comment by rogerxsj** [comment]
@MhLiao Dear author:
I am trying to pretrain the dbnet with synthtext-800k, could you provide me the *.yaml file as a reference？ Thanks a lot！

**Comment by MhLiao** [comment]
@rogerxsj The only difference is the two parameters `epochs` in the YAML file are both set to 400.
